publisher.rb

```diff
@@ -0,0 +1,27 @@
+require 'aws-sdk-sqs'
+require_relative 'load_aws'
+
+class Publisher
+  
+  attr_reader :sqs
+
+  def initialize
+    creds = Aws::Credentials.new(AWS["access_key_id"], AWS["secret_access_key"])
+    @sqs = Aws::SQS::Client.new(region: AWS["region"], credentials: creds)
+  end
+
+  # For Standard Queue
+  def create_std_q(queue_name)
+    std_queue = sqs.create_queue(queue_name: queue_name)
+  end
+
+  def publish_messages(queue_url, options=[])
+    delay_seconds = options[1].nil? ? 5 : options[1]
+    0.upto(Float::INFINITY) do |count|
+      message = { count: count, timestamp: Time.now.strftime("%GW%V%uT%H%M%S%L%z") }.to_json
+      puts "Sending message - #{message}"
+      sqs.send_message(queue_url: queue_url, message_body: message, delay_seconds: delay_seconds)
+      sleep(5)
```

Comment:
`s/5/delay_seconds/g`

Reply:
Delay seconds means make the message available for processing after the time specified..
DelaySeconds
The length of time, in seconds, for which to delay a specific message. Valid values: 0 to 900. Maximum: 15 minutes. Messages with a positive DelaySeconds value become available for processing after the delay period is finished. If you don't specify a value, the default value for the queue applies.
https://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_SendMessage.html
The queue should only send a message out every 5 seconds or the time between sending messages.
Use another term for L19 - set it to `sleep_period` and use that instead of `5` hence the substitute 5 with delay_seconds.
Removed `delay_seconds` from L23, its not needed.

Reply:
Ok got this. Do we want 5 seconds sleep period to be configurable or steady?

Reply:
Keeping it configurable with a max limit of 20 seconds.

Reply:
Use the regexp - given in #3